assembl/graphql/docstrings.py

```diff
@@ -0,0 +1,961 @@
+    default_preferences = """The default discussion preferences. These are server wide settings, independent of the debate."""
+    locales = """The list of locales supported on the debate. These are the languages of the debate."""
+    total_sentiments = """The total count of sentiments on the debate, regardless of chosen type. Deleted users' sentiments are not counted."""
+    has_syntheses = """A boolean flag indicating if the debate has yet released a synthesis or not."""
+    vote_session = """A vote session's meta data, if a vote session exists."""
+    resources = """A list of Resource meta data on the debate."""
+    resources_center = """A singular Resource Center meta data object."""
+    has_resources_center = """A boolean flag indicating if the debate has a resource center set or not."""
+    sections = """A list of Section meta data on the discussion."""
+    legal_contents = """The legal contents metadata representing the data."""
+    has_legal_notice = """A boolean flag of whether a debate has set a legal notice."""
+    has_terms_and_conditions = """A boolean flag of whether a debate has set a terms and conditions page."""
+    has_cookies_policy = """A boolean flag of whether a debate has set a cookie policy page."""
+    has_privacy_policy = """A boolean flag of whether a debate has set a privacy policy page."""
+    visits_analytics = """The object containing the summary data of analytics on the page, based on time-series analysis of analytics engine data."""
+    discussion = """The discussion object metadata."""
+    landing_page_module_types = """The metadata object for LandingPageModule object."""
+    landing_page_modules = """A list of LandingPageModules."""
+    text_fields = """A list of ConfigurableField union, where each text field represents a field on a bound entity."""
+    profile_fields = """A list of ConfigurableField union, where each text field represents a field on a profile only."""
+    timeline = """A list of DiscussionPhase objects, descriping the timeline objects on the debate."""
+
+
+class Discussion:
+    __docs__ = """The Discussion object. This object describes certain parts of the core state of the debate."""
+    id = Default.object_id % ("Discussion",)
+    homepage_url = """A URL for the homepage (optional). Often placed on the logo."""
+    title = """The title of the discussion, in the language specified by the input"""
+    subtitle = """The subtitle of the discussion, in the language specified by the input"""
+    button_label = """The value inside of the participation button in the landing page."""
+    header_image = Default.document % ("The file representing the header of the landing page. ", )
+    logo_image = Default.document % ("The file representing the logo of the debate. ", )
+
+
+class UpdateDiscussion:
+    __doc__ = """The mutation that allows to update an existing Discussion object"""
```

Comment:
You can just write the docstring on the class, without `__doc__ = `. The string will be available on the `__doc__` class attribute.

```
>>> class A(object):
...   """lalala"""
... 
>>> A.__doc__
'lalala'
```

Reply:
Yes, absolutely. The reason for this was to keep the editable string of text for document in one place as we one-shot this. Also, for future, any non-dev or front-end devs can edit documentation without python knowledge.